tests/trinity/integration/trinity_test_plugin/trinity_test_plugin.py

```diff
@@ -0,0 +1,5 @@
+class TestPlugin():
```

Comment:
i think I'm +1 on including these files, but for installation I think I'd prefer we pre-build the package and install directly from the `tar.gz` or `wheel`.  You should be able to build these using `python setup.py sdist` or `python setup.py bdist_wheel`.

Then in the `tox.ini` under the `trinity-integration` test suite you'll need to do a little discovery but I think you should be able to just add to the `deps` line [here](https://github.com/ethereum/py-evm/blob/master/tox.ini#L68) add the file path to the built package.

So we only really need to test that the plugin gets picked up (and probably a small sanity check that shows that the test package is actually installed) which you can do with `pkg_resources.get_distribution('trinity_test_plugin')`

Reply:
@ankitchiplunkar If I'm reading this correctly, this should solve your current problem where the tests are flaky when they include the installation part. Because with this, the installation part would already be handled by `tox` before the test runs.

Reply:
pinging @ankitchiplunkar Are you planning to continue with this?

Reply:
@cburgdorf yes got stuck with something, will pick it again in the weekend.